### ISSUE TITLE
regression: scalapb fails to compile

### ISSUE THREAD

**Issue by SethTisue** [scala/community-builds#562]
seen at e.g. https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-community-build/1608/consoleFull

```
[scalapb] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.7-RC1/project-builds/scalapb-5f3ef2e656e2083e33bb690423f33e9c3d470a9e/scalapb-runtime/shared/src/main/scala/com/trueaccord/scalapb/GeneratedExtension.scala:102: value zip is not a member of com.trueaccord.lenses.Lens[scalapb.UnknownFieldSet.Field,Seq[E]]
[scalapb] [error]         .compose(listLens zip UnknownFieldSet.Field.lengthDelimitedLens)
[scalapb] [error]                           ^
[scalapb] [error] one error found
```

fyi @thesamet @xuwei-k

**Comment by SethTisue** [comment]
dealt with it by freezing scalapb in 10b0a3ce5319c3189f534feddeb7d6533f131ade

**Comment by xuwei-k** [comment]
The last commit of scalapb/Lenses v0.4.12 is https://github.com/scalapb/Lenses/commit/301aafa6552ec81cf0 (Add support for zipping two lenses) but  current master branch does not contains it.
I guess @thesamet forget push some commits to master branch.

**Comment by thesamet** [comment]
Yes, the release processes includes a push to github and it appears that that didn't happen for some reason. Will fix later and then submit PR to revert 10b0a3c

**Comment by SethTisue** [comment]
> Will fix later and then submit PR to revert 10b0a3c

took care of it in https://github.com/scala/community-builds/pull/605
